lago/export.py

```diff
@@ -243,14 +257,98 @@ def export(self):
                     shutil.rmtree, self.dst, ignore_errors=True
                 )
                 self.copy()
-                self.sparse()
+                if not self.disk['format'] == 'iso':
+                    self.sparse()
                 self.calc_sha('sha1')
                 self.update_lago_metadata()
                 self.write_lago_metadata()
                 self.compress()
                 rollback.clear()
 
 
+class VMExportManager(object):
+    """
+    :class:VMExportManager object is responsible on the export process of a
+    list of disks.
+
+    Attributes:
+        disks (list of dicts): Disks to export.
+        dst (str): Where to place the exported disks.
+        compress(bool): If True compress each exported disk.
+        with_threads(bool): If True, run the export in parallel
```

Comment:
Maybe indicate that it will run a thread per disk. Also, if its not complicated might change it to an int so that:
threads=0 - > run a thread for each disk
threads=N -> run N threads.

Can be done later of course.